packages/api/lambdas/sf-event-sqs-to-db-records.js

```diff
@@ -77,30 +83,39 @@ const hasNoAsyncOpOrExists = async (cumulusMessage, knex) => {
   }, knex, tableNames.asyncOperations);
 };
 
-const getMessageCollection = async (cumulusMessage, knex) => {
+const getMessageCollectionCumulusId = async (cumulusMessage, knex) => {
```

Comment:
Yeah, the `message` package is just helpers for parsing info out of a Cumulus message. I could put this in `db`, though I wasn't sure it would get used outside of this Lambda

Reply:
I'm fine with YAGNI as a judgement call here, the concern really is maintainability situations where we don't realize this is here for whatever reason and re-implementing it.